conceptnet5/vectors/cli.py

```diff
@@ -98,16 +102,34 @@ def run_intersect(input_filenames, output_filename, projection_filename):
 @cli.command(name='evaluate')
```

Comment:
I think we should name this command "evaluate_wordsim" instead of "evaluate", since now we have "evaluate_analogies".